src/get-started/flutter-for/web-devs.md

```diff
@@ -863,10 +863,10 @@ with some formatting characteristics.
 To display text that uses multiple styles
 (in this example, a single word with emphasis),
 use a [`RichText`][]widget instead.
-Its `text` property can specify one or more [`TextSpan`][] widgets
+Its `text` property can specify one or more [`TextSpan`][]
```

Comment:
`TextSpan` should probably have a trailing s in this context because it is potentially many TextSpans.